tox.ini

```
@@ -16,7 +16,8 @@ python =
 deps =
     dj22: django>=2.2,<2.3
     dj31: django>=3.1,<3.2
-    dj32: django>=3.2,<3.3
+    dj32: django>=3.2,<4.0
```

Let's keep it `django>=3.2,<3.3`
```suggestion
    dj32: django>=3.2,<3.3
```

---

ok, even if according django versionning 3.3 will never exists